pkg/registration/register/aws_irsa/aws_irsa_test.go

```diff
@@ -230,7 +230,7 @@ func TestIsHubKubeConfigValidFunc(t *testing.T) {
 	}
 	for _, c := range cases {
 		t.Run(c.name, func(t *testing.T) {
-			driver := NewAWSIRSADriver()
+			driver := NewAWSIRSADriver("", "")
```

Comment:
Please ignore this one, we are planning to fix this in future, when we implement the completion of joining, Did this now to resolve a compilation error.